kidash/kidash.py

```diff
@@ -789,8 +800,7 @@ def get_index_pattern_json(elastic, index_pattern_id):
 def get_search_from_vis(elastic, vis):
     search_id = None
     vis_json = get_vis_json(elastic, vis)
-    if not vis_json:
-        search_id
```

Comment:
Also, removed this completely as this seems to have no effect.

Reply:
Makes sense.